### ISSUE TITLE
Speaking page adjustment

### ISSUE THREAD

**Issue by vavroom** [nicolas-steenhout/main#37]
1. The "hire nic for your next engagement" text seems so big it's hard to read. I could see an auditor flagging this as "if it's got such visual weight, it should be a heading". More importantly, I don't think it's catching the attention and comprehension of the readers. It'll be hard to parse and hard to act upon.
2. The keynote and presentation topics section - could we make that with the dark blue background, a bit like on the homepage? Just to add variation on the page?
3. I'll be adding some photos of me speaking. Probably 2 or 3. Could you please add placeholders for those?

**Comment by digilou** [comment]
closed #46
